src/classes/RD_AddDonationsBTN_CTRL.cls

```diff
@@ -148,8 +148,12 @@ public with sharing class RD_AddDonationsBTN_CTRL {
         } else if (sc.getId() == null) {
             addMessageAndReturn(ApexPages.Severity.ERROR, System.Label.RD_ErrorAddDonationMissingId);
 
-        } else if (!Schema.SObjectType.Opportunity.isCreateable()) {
-            addMessageAndReturn(ApexPages.Severity.INFO, System.Label.RD_ErrorAddDonationPermissionDenied);
+        } else if (!Schema.SObjectType.Opportunity.isCreateable()
```

Comment:
Can we use `UTIL_Permissions` to verify the CRUD on `Opportunity`? If so, see example of how to use the test class stub to test various conditions.

Reply:
Even, I thought about it, will change it.